guide/installation/gmc_install/gmc_install.md

```diff
@@ -0,0 +1,118 @@
+# GenAI-microservices-connector(GMC) Installation
+
+This document will introduce the GenAI Microservices Connector (GMC) and its installation. It will then use the ChatQnA pipeline as a use case to demonstrate GMC's functionalities.
+
+## GenAI-microservices-connector(GMC)
+
+GMC can be used to compose and adjust GenAI pipelines dynamically on Kubernetes. It can leverage the microservices provided by GenAIComps and external services to compose GenAI pipelines. External services might be running in a public cloud or on-prem by providing an URL and access details such as an API key and ensuring there is network connectivity. It also allows users to adjust the pipeline on the fly like switching to a different Large language Model(LLM), adding new functions into the chain(like adding guardrails),etc. GMC supports different types of steps in the pipeline, like sequential, parallel and conditional. For more information: https://github.com/opea-project/GenAIInfra/tree/main/microservices-connector
+
+
+
+## Install GMC
+
+**Prerequisites**
+
+- A running Kubernetes cluster with at least 16 CPUs, 32GB of memory, and 100GB of disk space. To install an Kubernetes cluster as your platform:
+[Kubernetes installation](../k8s_install/) 
+
+**Build and push your image to the location specified by `CTR_IMG`:**
+
+```sh
+make docker.build docker.push CTR_IMG=<some-registry>/gmcmanager:tag
+```
+
+**NOTE:** This image ought to be published in the personal registry you specified.
+And it is required to have access to pull the image from the working environment.
+Make sure you have the proper permission to the registry if the above commands don’t work.
+
+**Install GMC CRD**
+
+```sh
+kubectl apply -f config/crd/bases/gmc.opea.io_gmconnectors.yaml
+```
+
+**Get related manifests for GenAI Components**
+
+```sh
+mkdir -p $(pwd)/config/manifests
+cp $(dirname $(pwd))/manifests/ChatQnA/*.yaml -p $(pwd)/config/manifests/
+cp $(dirname $(pwd))/manifests/DocSum/xeon/docsum_llm.yaml $(pwd)/config/manifests/
+cp $(dirname $(pwd))/manifests/DocSum/gaudi/docsum_gaudi_llm.yaml $(pwd)/config/manifests/
+```
+
+**Copy GMC router manifest**
+
+```sh
+cp $(pwd)/config/gmcrouter/gmc-router.yaml -p $(pwd)/config/manifests/
+```
+
+**Create Namespace for gmcmanager deployment**
+
+```sh
+export SYSTEM_NAMESPACE=system
+kubectl create namespace $SYSTEM_NAMESPACE
+```
+
+**NOTE:** The `SYSTEM_NAMESPACE` should keep the same with the namespace defined in gmc-manager.yaml and gmc-manager-rbac.yaml.
```

Comment:
s/The `SYSTEM_NAMESPACE` should keep the same with the namespace defined in gmc-manager.yaml and gmc-manager-rbac.yaml.The `SYSTEM_NAMESPACE` should keep the same with the namespace defined in gmc-manager.yaml and gmc-manager-rbac.yaml.
/Please the exact same SYSTEM_NAMESPACE value setting you used while deploying gmc-manager.yaml and gmc-manager-rbac.yaml.